tests/test_structural-variantcalling.yml

```diff
@@ -0,0 +1,14 @@
+- name: structural-variantcalling
+  tags:
+    - integration
```

Comment:
This doesn't use the docker config, right? So the `no_docker_conf` tag should be added.